src/templates/blog-post.js

```diff
@@ -124,6 +124,12 @@ const BlogPost = ({ data }) => {
             <meta property="og:image" content={imageUrl} />
             <meta property="og:description" content={subtitle} />
             <meta property="og:type" content="article" />
+            <meta name="twitter:card" content="summary" />
```

Comment:
@ric9176 why don't you use your here your function createSocialMetas?

Reply:
This one is a bit different with what's passed, here we do `<meta name="twitter:creator" content={`@${authorTwitter}`} /> while "twitter:site" is @reactgqlacademy also the "og:type" must be "article" rather than "website". Basically this one is not generic as it's our blog articles and it's the most important one for these social metas. I will check in with Milos that it all looks good. 
`

Reply:
👍